libs/drupal_acces.js

```diff
@@ -0,0 +1,136 @@
+var request = require('superagent');
+var logger = require('../libs/logger')(module);
+
+var DRUPAL_PROFILE_API_URL = process.env.DRUPAL_PROFILE_API_URL;
+var DRUPAL_PROFILE_API_USER = process.env.DRUPAL_PROFILE_API_USER;
+var DRUPAL_PROFILE_API_PASSWORD = process.env.DRUPAL_PROFILE_API_PASSWORD;
+
+function _parse_fron_anchor_tag(uid_string) {
+    var re = /<a.*>(.+)<\/a>/g;
+    m = re.exec(uid_string);
+    re.lastIndex = 0;
+    if (m.length === 2) {
+        return m[1];
+    } else {
+        return undefined;
+    }
+}
+
+function _parse_user(res_body) {
+    if (res_body.length === 0) {
+        return undefined
+    }
+    has_ticket_ = res_body[0]['PHP'] !== "";
+    return {
+        last_name: res_body[0]['Last name'],
+        first_name: res_body[0]['First name'],
+        uid: res_body[0]['uid'],
+        email: _parse_fron_anchor_tag(res_body[0]['E-mail']),
+        phone: res_body[0]['Phone number'],
+        has_ticket: has_ticket_
+    };
+}
+var login = function (forceLogin) {
```

Comment:
this first function is there to create a closure to keep the `request_agent` it has no params as its execution is happening right after definition (line 55)